### DISCUSSION TITLE
can't access by education benefits.

### DISCUSSION THREAD

**Discussion by arijitbarai** [community/community#59079] [GitHub Education]
### Hi everyone,

I don't know what happened but I can access the global campus and my benefits. my application got approve on 17 june and i was able to use my benefits from 20 but for some reason  today i can't access them

**Comment by leonden** [comment]
When I wanted to apply for GitHub education, I simply registered an account with my education email. It was as easy as that 😄

**Comment by samus-aran** [comment]
Hi @arijitbarai  👋

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.

We apologize for any inconvenience this may cause.

To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Thank you for your understanding.

**Reply by arijitbarai** [reply]
@samus-aran  sorry but my university don't provide students edu mail, and i can't pass the first page without edu mail so i didn't able to access the page where i can upload my documents

**Reply by arijitbarai** [reply]
and i already has create a ticket about this 3 days ago but didn't get any replay 
#2268295
